src/server/config.rs

```diff
@@ -315,6 +375,21 @@ fn validate_label(s: &str, tp: &str) -> Result<()> {
     Ok(())
 }
 
+pub struct ServerConfigManager(pub Arc<RwLock<Config>>);
+
+impl ConfigManager for ServerConfigManager {
+    fn dispatch(
+        &mut self,
+        change: ConfigChange,
+    ) -> std::result::Result<(), Box<dyn std::error::Error>> {
+        info!(
+            "server config changed";
+            "change" => ?change,
+        );
+        Ok(())
```

Comment:
I believe you need to handle the new configuration here?

Reply:
Yeah, Is there any special handling we need to do? I noticed the updated value was saved, though I couldn't figure out why.

Reply:
The updated value was saved to the config file to prevent lost updates after a restart. I think handling it just like how `BackupConfig` does will be okay.